index.html

```diff
@@ -265,11 +265,10 @@ <h3>
           Every [=permission=] has a <dfn class="export" data-dfn-for="permission">lifetime</dfn>,
           which is the duration for which a particular permission remains [=permission/granted=]
           before it reverts back to its [=permission/default state=]. A [=permission/lifetime=]
-          could be until a particular [=ECMAScript/Realm=] is destroyed, until a particular
```

Comment:
We should probably still allow this at least until we replace lifetime with a keying-like solution?

Reply:
Reverted it.